src/main/java/org/jenkinsci/test/acceptance/plugins/ssh_slaves/SshSlaveLauncher.java

```diff
@@ -17,14 +17,15 @@
  */
 @Describable("hudson.plugins.sshslaves.SSHLauncher")
 public class SshSlaveLauncher extends ComputerLauncher {
-    public final Control host = control("host");
-    public final Control credentialsId = control("credentialsId");
-    public final Control prefixCmd = control("prefixStartSlaveCmd");
-    public final Control suffixCmd = control("suffixStartSlaveCmd");
-    public final Control timeout = control("launchTimeoutSeconds");
-    public final Control retries = control("maxNumRetries");
-    public final Control javaPath = control("javaPath");
-    public final Control jvmOptions = control("jvmOptions");
+    public final Control host = control("/host");
+    public final Control credentialsId = control("/credentialsId");
+    public final Control prefixCmd = control("/prefixStartSlaveCmd");
+    public final Control suffixCmd = control("/suffixStartSlaveCmd");
+    public final Control timeout = control("/launchTimeoutSeconds");
+    public final Control retries = control("/maxNumRetries");
+    public final Control javaPath = control("/javaPath");
+    public final Control jvmOptions = control("/jvmOptions");
+    public final Control sshHostKeyVerificationStrategy = control("/"); //weird path because of JENKINS-43076
```

Comment:
so otherwise I needed to use either  control("")  for the latest which 1) I think it's even more confusing 2) causes issues when resolving the path.

An alternative was to use control("/") only for the latest as it is now, but then it would have had a different format compared with the others. Plus other classes (e.g. DumSlave) use this approach.. So I can roll back, but I think it's clearer this way than having host, javaPath and then a "/"